<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>391</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="lstDispositivos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>256</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Electrodom√©sticos conectados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
